Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -907,27 +909,35 @@ - (UITableViewCell *)tableView:(__attribute__((unused)) UITableView *)tableView
                                      initWithStyle:UITableViewCellStyleDefault
                                      reuseIdentifier:nil];
       cell.selectionStyle = UITableViewCellSelectionStyleNone;
-      self.barcodeLabelImage = [[UILabel alloc] initWithFrame:CGRectMake(0, 20, self.view.bounds.size.width, 140)];
-      self.barcodeLabelImage.text = [NSString stringWithFormat:@"A%@B", [NYPLAccount sharedAccount:self.accountType].authorizationIdentifier];
-      self.barcodeLabelImage.font = [UIFont fontWithName:@"CodabarLarge" size:36.0];
-      self.barcodeLabelImage.textAlignment = NSTextAlignmentCenter;
-      self.barcodeLabelImage.adjustsFontSizeToFitWidth = YES;
-      UITapGestureRecognizer *tap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(barcodeZoom)];
-      
-      [self.barcodeLabelImage addGestureRecognizer:tap];
-      self.barcodeLabelImage.userInteractionEnabled = YES;
 
-      [cell.contentView addSubview:self.barcodeLabelImage];
-      
-      UILabel *labelD = [[UILabel alloc] initWithFrame:CGRectMake(0, 95, self.view.bounds.size.width, 10)];
-      labelD.text =  [NYPLAccount sharedAccount:self.accountType].authorizationIdentifier;
-      labelD.font = [UIFont systemFontOfSize:10];
-      labelD.textAlignment = NSTextAlignmentCenter;
-      labelD.adjustsFontSizeToFitWidth = YES;
-      [cell.contentView addSubview:labelD];
-
-      
 
+      if (![self librarySupportsBarcodeDisplay]) {
+        NYPLLOG(@"A nonvalid library was attempting to create a barcode image.");
+      } else {
+        UIImage *barcodeImage = [NYPLBarcode imageFromString:@"23333103390991"
```

Comment:
what's 23333103390991?

Reply:
This was a mistake! Thank you! A lot of people would have been using my library card...